services/user.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username,
+                password,
+                email=None,
+                first_name=None,
+                last_name=None):
+    user = User.objects.create_user(username=username, password=password,)

```

Comment:
Taras is right. When you need to use User model - use either `settings.AUTH_USER_MODEL` or `get_user_model()`.